pkg/universe.dagger.io/package.json

```
@@ -1,7 +1,8 @@
 {
   "license": "Apache-2.0",
   "scripts": {
-    "test": "bats --report-formatter junit --print-output-on-failure --jobs 4 $(find ${TESTDIR:-.} -type f -name '*.bats' -not -path '*/node_modules/*')"
+    "test": "bats --report-formatter junit --print-output-on-failure --jobs 4 $(find ${TESTDIR:-.} -type f -name '*.bats' -not -path '*/node_modules/*' -not -path '*/x/*')",
+    "test-experimental": "bats --report-formatter junit --print-output-on-failure --jobs 4 $(find ${TESTDIR:-.} -type f -name '*.bats' -not -path '*/node_modules/*')"
```

```suggestion
    "test-experimental": "bats --report-formatter junit --print-output-on-failure --jobs 4 $(find ${TESTDIR:-./x} -type f -name '*.bats' -not -path '*/node_modules/*')"
```

We should only run the experimental tests as part of this step (right now if you look in CI it includes all of them in this step)